source/reference/modules/studio/administration.rst

```diff
@@ -1131,3 +1134,139 @@ on the dropdown arrow in the ``Project`` field.  Give it a good ``Project Name``
+|
+
+^^^^^^^^^
+API Token
+^^^^^^^^^
+To create a new access token, click on ``Token Management`` from the Main Menu, then click on the ``API Token`` button.
+The only required field for the access token is the label to identify it, however, it is also recommended to set
+an expiration date to minimize the risk of lost or stolen tokens being used without being noticed.
+
+.. figure:: /_static/images/jwt/create-token.webp
+    :width: 70%
+    :alt: Crafter Studio - Create API Access Token
+    :align: center
+
+|
+
+Once the expiration date is reached the access token will stop working automatically. Click on the ``Submit`` button to
+create the token.
+
+.. figure:: /_static/images/jwt/create-token-2.webp
+    :width: 70%
+    :alt: Crafter Studio - Access Token Expiration
+    :align: center
+
+|
+
+The next step is to copy the value of the access token. The value of the access token will not be stored on the server,
+so it needs to be stored by the user in a safe place as it is impossible to recover it after it is created.
+
+.. figure:: /_static/images/system-admin/main-menu/access-token-created.webp
+    :width: 70%
+    :alt: Crafter Studio - Access Token Created
+    :align: center
+
+|
+
+If an access token is lost or exposed in any way it should be disabled or completely deleted to avoid any
+possible use. To delete a token, simply click on the trash can icon to the right of the token you want to delete.
+
+.. figure:: /_static/images/system-admin/main-menu/delete-token-1.webp
+    :width: 70%
+    :alt: Crafter Studio - Delete a Token
+    :align: center
+
+|
+
+You can also delete multiple tokens at once by placing a checkmark on the tokens you want to delete, then clicking on
+``Delete Selected``.
+
+.. figure:: /_static/images/system-admin/main-menu/delete-token-2.webp
+    :width: 70%
+    :alt: Crafter Studio - Delete Multiple Tokens
+    :align: center
+
+|
+
+To disable/enable a token, simply click on the slider on the right side of the token next to the trash can icon.
+
+.. figure:: /_static/images/system-admin/main-menu/token-disable.webp
+    :width: 70%
+    :alt: Crafter Studio - Disable/Enable Token
+    :align: center
+
+|
+
+For an example of how to use the generated API token, see :ref:`crafter-cli`.
+
+.. note:: Users needs the ``manage_access_token`` permission to create access tokens
+
+^^^^^^^^^^^^^
+Preview Token
+^^^^^^^^^^^^^
+.. version_tag::
+    :label: Since
+    :version: 4.2.0
+
+To create a Preview Token, click on ``Token Management`` from the Main Menu, then click on the ``Preview Token`` button.
+
+.. figure:: /_static/images/system-admin/main-menu/create-preview-token.webp
+    :width: 70%
+    :alt: Crafter Studio - Create Preview Access Token
+    :align: center
+
+|
+
+The only required fields for the preview token is the dropdown for selecting projects to grant preview access, and to set
+an expiration date to minimize the risk of lost or stolen tokens being used without being noticed. Click on the
```

Comment:
The date is required too; it is just pre-populated to 1 month ahead of the present date.